runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -88,21 +88,48 @@ J9::ClassEnv::classFlagsValue(TR_OpaqueClassBlock * classPointer)
    {
    if (auto stream = TR::CompilationInfo::getStream())
       {
+      uintptrj_t classFlags = 0;
```

Comment:
This code is not needed. If the JIT asks for generic flags we should get the most recent version because we don't know if they have changed.

Reply:
Thank you, I have changed this section of code. `J9::ClassEnv::classFlagsValue` now obtains class flags from remote call.